src/main/java/org/generationcp/middleware/api/brapi/v2/attribute/AttributeMapper.java

```diff
@@ -0,0 +1,36 @@
+package org.generationcp.middleware.api.brapi.v2.attribute;
+
+import org.generationcp.middleware.service.api.study.VariableDTO;
+
+import java.util.List;
+
+public class AttributeMapper {
```

Comment:
We can move this to BMSAPI (related to comment about making VariableServiceBrapi services generic)